calico/felix/futils.py

```diff
@@ -482,3 +482,14 @@ def iso_utc_timestamp():
     time_now = datetime.utcnow()
     time_formatted = time_now.replace(microsecond=0).isoformat() + 'Z'
     return time_formatted
+
+
+def find_set_bits(mask):
+    """Generates an integer for each set bit in the input.
```

Comment:
Worth adding that (1) that integer is the value of the relevant bit - as opposed to e.g. its bit position; (2) bits are returned in order of least significance.